[2025-09-12T06:19:10.338+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-12T06:19:08.575440+00:00 [queued]>
[2025-09-12T06:19:10.341+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-12T06:19:08.575440+00:00 [queued]>
[2025-09-12T06:19:10.343+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-12T06:19:10.347+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_kafka_consumer> on 2025-09-12 06:19:08.575440+00:00
[2025-09-12T06:19:10.348+0000] {standard_task_runner.py:60} INFO - Started process 2568 to run task
[2025-09-12T06:19:10.350+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'aviationstack_kafka_dag', 'run_kafka_consumer', 'manual__2025-09-12T06:19:08.575440+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/aviationstack_dag.py', '--cfg-path', '/tmp/tmpwby2_tgk']
[2025-09-12T06:19:10.351+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask run_kafka_consumer
[2025-09-12T06:19:10.370+0000] {task_command.py:423} INFO - Running <TaskInstance: aviationstack_kafka_dag.run_kafka_consumer manual__2025-09-12T06:19:08.575440+00:00 [running]> on host e2b24969dd82
[2025-09-12T06:19:10.395+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='aviationstack_kafka_dag' AIRFLOW_CTX_TASK_ID='run_kafka_consumer' AIRFLOW_CTX_EXECUTION_DATE='2025-09-12T06:19:08.575440+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-12T06:19:08.575440+00:00'
[2025-09-12T06:19:10.396+0000] {logging_mixin.py:188} INFO - Starting consumer...
[2025-09-12T06:19:10.397+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.4', 9092)]>: connecting to kafka:9092 [('172.18.0.4', 9092) IPv4]
[2025-09-12T06:19:10.398+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Connection complete.
[2025-09-12T06:19:10.398+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Closing connection. 
[2025-09-12T06:19:10.706+0000] {cluster.py:401} INFO - Coordinator for group/aviation_flight_data_test_group_02 is ('coordinator-1', 'kafka', 9092, None)
[2025-09-12T06:19:10.707+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group aviation_flight_data_test_group_02
[2025-09-12T06:19:10.707+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-09-12T06:19:10.707+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group aviation_flight_data_test_group_02
[2025-09-12T06:19:10.707+0000] {base.py:419} INFO - Failed to join group aviation_flight_data_test_group_02: NodeNotReadyError: coordinator-1
[2025-09-12T06:19:10.708+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.4', 9092)]>: connecting to kafka:9092 [('172.18.0.4', 9092) IPv4]
[2025-09-12T06:19:10.708+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.4', 9092)]>: Connection complete.
[2025-09-12T06:19:10.812+0000] {base.py:532} INFO - (Re-)joining group aviation_flight_data_test_group_02
[2025-09-12T06:19:10.823+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-83f23bbb-b311-4114-a970-1c09fbb511bf for group aviation_flight_data_test_group_02; will retry join-group
[2025-09-12T06:19:10.823+0000] {base.py:419} INFO - Failed to join group aviation_flight_data_test_group_02: [Error 79] MemberIdRequiredError
[2025-09-12T06:19:10.823+0000] {base.py:532} INFO - (Re-)joining group aviation_flight_data_test_group_02
[2025-09-12T06:19:16.854+0000] {base.py:594} INFO - Successfully joined group aviation_flight_data_test_group_02 <Generation 1 (member_id: kafka-python-2.2.15-83f23bbb-b311-4114-a970-1c09fbb511bf, protocol: range)>
[2025-09-12T06:19:16.854+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-09-12T06:19:16.867+0000] {subscription_state.py:254} INFO - Updated partition assignment: []
[2025-09-12T06:19:16.868+0000] {consumer.py:249} INFO - Setting newly assigned partitions () for group aviation_flight_data_test_group_02
[2025-09-12T06:19:20.399+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-12T06:19:20.410+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=aviationstack_kafka_dag, task_id=run_kafka_consumer, execution_date=20250912T061908, start_date=20250912T061910, end_date=20250912T061920
[2025-09-12T06:19:20.427+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-12T06:19:20.440+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
